lib/offsite_payments/integrations/world_pay.rb

```diff
@@ -230,6 +231,12 @@ def custom_params
           return @custom_params ||= read_custom_params
         end
 
+        # Check if the request comes from IP range 195.35.90.0 – 195.35.91.255
+        def valid_sender?(ip)
+          return true if OffsitePayments.mode == :test 
+          IPAddr.new("195.35.90.0/23").include?(IPAddr.new(ip))
```

Comment:
These guys really have *that* many IPs? I know this works, but I wish we didn't have to allow 500+ addresses